lib/kadm5/ipropd_master.c

```diff
@@ -1519,6 +1563,7 @@ static char sHDB[] = "HDBGET:";
 static char *realm;
 static int version_flag;
 static int help_flag;
+static int restarter_flag;
```

Comment:
Separate commit for this?

Reply:
Sure.  I only needed this for debugging though -- it's so much easier to debug if you don't have to follow fork and all that.